### PR TITLE
added resource group script

### PR DESCRIPTION
**Pull request by chethanamaidam**

### CODE REVIEW

**Comment by chethanamaidam** [comment]
hello,

please review my pr and let me know further

File: Main/main.tf [hunk 1]
```diff
@@ -0,0 +1,10 @@
+provider "azurerm" {
+    features {}
+}
+
+resource "azurerm_resource_group" "resource_group" {
+      name        =  local. rg_name
```
Comment (techslateramu) [inline Main/main.tf]: why is there extra space, here ?

File: Main/terraform.tfstate [hunk 2]
```diff
@@ -0,0 +1,38 @@
+{
```
Comment (techslateramu) [inline Main/terraform.tfstate]: state file shouldn't be checked in .. you can remove this from here

**Review by techslateramu — COMMENTED** [review]
Please fix the comments  !

File: Main/main.tf (hunk 1 above)
Comment (chethanamaidam) [reply Main/main.tf]: ok I will fixit now

File: Main/terraform.tfstate (hunk 2 above)
Comment (chethanamaidam) [reply Main/terraform.tfstate]: okay I will remove

**Comment by chethanamaidam** [comment]
hello

based on your comments fixed them you can review again

**Comment by chethanamaidam** [comment]
can you please review now ??

**Review by techslateramu — APPROVED** [review]
Good. Approving now

**Pull request closed without merging**